coral/src/app/features/topics/details/components/TopicDetailsHeader.tsx

```diff
@@ -28,6 +28,18 @@ function TopicDetailsHeader(props: TopicOverviewHeaderProps) {
     topicExists,
   } = props;
 
+  // The call to getConsumerOffsets in TopicDetails needs an environmentId
+  // But on first load, environmentId is undefined
+  // So we need to set it in this way, to be able to fetch correctly
+  // Ideally, environmentId should be available on first load
+  // Because this setEnvironmentId will trigger a superfluous refetch of getTopicOverview
+  // @TODO: find a better way to have the initial env of the Topic overview be the lowest env, instead of undefined
```

Comment:
We could also have the `getTopicOverview` call in the `/pages/topics/details/index.tsx` component, and pass the initial data as prop to `TopicDetails`, but I'm not sure we want to go into this pattern 🤔